test/test-container-start-command.js

```diff
@@ -6,14 +6,16 @@
 var Container = require('../lib/drivers/direct/container');
 var tap = require('tap');
 
-var basicOpts = {
-  instanceId: '_id',
-  baseDir: '_base',
-  server: {},
-};
+function newContainer() {
+  return new Container({
+    instanceId: '_id',
+    baseDir: '_base',
+    server: {},
```

Comment:
no, they are mandatory params that need some value in them, don't care what, just so we can have a Container and use it in the tests